crypto/hash.go

```diff
@@ -118,22 +118,21 @@ func HashArgsSuite(suite abstract.Suite, args ...interface{}) ([]byte, error) {
 }
 
 // ConvertToBinaryMarshaler takes a slice of interfaces and returns
-// a slice of BinaryMarshalers.
-func convertToBinaryMarshaler(args ...interface{}) ([]encoding.BinaryMarshaler, error) {
+// a slice of BinaryMarshalers
+func ConvertToBinaryMarshaler(args ...interface{}) ([]encoding.BinaryMarshaler, error) {
```

Comment:
We will move all of the stuff from `crypto/hash.go` to the crypto library at some point. So I this can be addressed later.